portal-cdk/portal_cdk/portal_cdk_stack.py

```diff
@@ -184,6 +212,24 @@ def __init__(
                 cache_policy=cloudfront.CachePolicy.CACHING_DISABLED,
                 response_headers_policy=cloudfront.ResponseHeadersPolicy.CORS_ALLOW_ALL_ORIGINS_WITH_PREFLIGHT_AND_SECURITY_HEADERS,
             ),
+            additional_behaviors={
+                "/error*.html": cloudfront.BehaviorOptions(
+                    origin=origins.S3Origin(error_bucket),
+                    origin_request_policy=cloudfront.OriginRequestPolicy.ALL_VIEWER_EXCEPT_HOST_HEADER,
+                    viewer_protocol_policy=cloudfront.ViewerProtocolPolicy.ALLOW_ALL,
+                    allowed_methods=cloudfront.AllowedMethods.ALLOW_GET_HEAD,
+                    cache_policy=cloudfront.CachePolicy.CACHING_DISABLED,
+                    response_headers_policy=cloudfront.ResponseHeadersPolicy.CORS_ALLOW_ALL_ORIGINS_WITH_PREFLIGHT,
+                ),
+            },
+            error_responses=[
+                cloudfront.ErrorResponse(
+                    http_status=503,
```

Comment:
503 is good. I manually borked the lambda and it returned 500, so we should add 500. But 503 is definitely still a real thing. So, 503 & 500.